Missing implementation of getTemperature()

The library declares a `BoschSensorClass::getTemperature()` function:

https://github.com/arduino-libraries/Arduino_BMI270_BMM150/blob/aabdaf680567cffc1721ffbd0baf76cfc7dcd62a/src/BoschSensorClass.h#L67

But if you try to use it:

```cpp
#include <Arduino_BMI270_BMM150.h>
void setup() {
  IMU.getTemperature();
}
void loop() {}
```

compilation fails due to the declared function not being implemented:

```text
C:\Users\per\AppData\Local\Temp\.arduinoIDE-unsaved2023718-18988-8022i1.jjmfp\sketch_aug18a/sketch_aug18a.ino:3: undefined reference to `BoschSensorClass::getTemperature()'
```

I see from its datasheet that the sensor does support temperature readings.

--- a/src/BoschSensorClass.h
+++ b/src/BoschSensorClass.h
@@ -64,8 +64,6 @@ class BoschSensorClass {
     virtual int magneticFieldAvailable(); // Number of samples in the FIFO.
     virtual float magneticFieldSampleRate(); // Sampling rate of the sensor.
 
-    float getTemperature();
-
   protected:
     // can be modified by subclassing for finer configuration
     virtual int8_t configure_sensor(struct bmm150_dev *dev);

